File: PRAK405_2310817210022_MUHAMMAD_ADH-DHIYA_US_SALIM.py
for i in range(3):
    n, k = map(int, input().split())
    SumAll = 0
    for i in range(1, n+1):
        SumLine = 0
        for j in range(i, 0, -1):
            h = j * k
            SumLine += h
            print(f"({j} * {k})", end="")
            if j > 1:
               print(" + ", end="")
        SumAll += SumLine
        print(f" = {SumLine}")
    print(SumAll)
    

File: PRAK405_2310817210022_MUHAMMAD_ADH-DHIYA_US_SALIM.c
#include <stdio.h>

int main() {
    int n, k;
    for(int i = 0; i < 3; i++){
    scanf("%d %d", &n, &k);

    int SumAll = 0;

        for (int i = 1; i <= n; i++) {
            int SumLine = 0;
            for (int j = i; j >= 1; j--) {
                int h = j * k;
                SumLine += h;
                printf("(%d * %d)", j, k, h);
                if (j > 1) {
                    printf(" + ");
                }
            }
            SumAll += SumLine;
            printf(" = %d\n", SumLine);
        }

        printf("%d\n", SumAll);
        }
        return 0;
    }